bridges/snowbridge/pallets/inbound-queue/fixtures/src/register_token.rs

```diff
@@ -7,12 +7,12 @@ use hex_literal::hex;
 use snowbridge_beacon_primitives::{
 	types::deneb, AncestryProof, BeaconHeader, ExecutionProof, VersionedExecutionPayloadHeader,
 };
-use snowbridge_inbound_queue_primitives::{EventProof, InboundQueueFixture, Log, Proof};
+use snowbridge_inbound_queue_primitives::{EventFixture, EventProof, Log, Proof};
```

Comment:
Can you revert the renaming please?

Reply:
I rename it because `InboundQueueFixture` will be reused when benchmarking the extrinsic `submit_delivery_receipt` for the outbound queue, so I prefer to make it more neutral.